dio/lib/src/transformers/utf8_json_transformer.dart

```diff
@@ -0,0 +1,184 @@
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:dio/dio.dart';
+import 'package:dio/src/compute/compute.dart';
+
+/// A [Transformer] that has a fast path for decoding utf8-encoded JSON.
```

Comment:
For sentences I think `utf8` should be `UTF-8`, so utf8-encoded can be `encoded in UTF-8`?

Reply:
yes, applied your suggestion